rskj-core/src/test/java/co/rsk/peg/BridgeSupportSvpTest.java

```diff
@@ -129,6 +130,10 @@ void setUp() {
             logs
         );
 
+        ECKey key = RskTestUtils.getEcKeyFromSeed("key");
+        RskAddress address = new RskAddress(key.getAddress());
+        when(rskTx.getSender(any())).thenReturn(address); // to not throw when calling update collections
```

Comment:
Should we explicitly say what this param should be?

Reply:
updated